tests/e2e/medic-collect.js

```diff
@@ -36,32 +36,31 @@ describe('medic-collect', () => {
     ]));
 
   /*  afterAll(() =>
-    Promise.all([ 'form:my_app_form', 'form:my_collect_form' ].map(id =>
+    await Promise.all([ 'form:my_app_form', 'form:my_collect_form' ].map(id =>
       db.get(id)
         .then((doc) => db.remove(doc)))));*/
 
   describe('without User-Agent header', () => {
-    it('is prompted for auth details if not supplied', () => {
+    it('is prompted for auth details if not supplied', async () => {
       // when
-      return rawHttpRequest(
+      const res = await rawHttpRequest(
         `HEAD /${dbName}/_design/medic/_rewrite/add?deviceID=imei%3A357578064823168 HTTP/1.1\r
 X-OpenRosa-Version: 1.0\r
 Date: ${new Date().toISOString()}\r
 Host: ${host}:${port}\r
 Connection: close\r
 \r\n`
-      ).then(res => {
+      );
         // then
-        assert.equal(res.statusCode, 401, JSON.stringify(res));
-        assert.equal(
-          res.headers['WWW-Authenticate'],
-          'Basic realm="Medic Mobile Web Services"',
-          JSON.stringify(res)
-        );
-      });
+      assert.equal(res.statusCode, 401, JSON.stringify(res));
+      assert.equal(
+        res.headers['WWW-Authenticate'],
+        'Basic realm="Medic Mobile Web Services"',
+        JSON.stringify(res)
+      );
     });
 
-    it('can fetch a list of forms', () => {
+    it('can fetch a list of forms', async () => {
```

Comment:
I appreciate that the test above was updated with async/await. This one seems to be a similar style, but has not been similarly updated. Is there something preventing that?